net/clamav/files/clamav.init

```diff
@@ -79,15 +79,15 @@ start_clamav_instance() {
 	echo "ScanPE " $ScanPE >> $CLAMD_CONFIGFILE
 	echo "DisableCertCheck " $DisableCertCheck >> $CLAMD_CONFIGFILE
 	echo "ScanELF " $ScanELF >> $CLAMD_CONFIGFILE
-	echo "DetectBrokenExecutables " $DetectBrokenExecutables >> $CLAMD_CONFIGFILE
```

Comment:
This looks like it can be aggregated into a single >> call.

Reply:
https://github.com/neheb/packages/commit/df0534bb702ec31ffcde027cb43e0d520bb12e08 is effectively what I meant.

I also meant single > call.

Reply:
Will take a look at it today, thanks! I would like to test it before cherry-picking it.

Reply:
Tested, thanks!